### PR TITLE
Fix status bar on WelcomeScreen

### PR DESCRIPTION
**Pull request by xboxones1**
The status bar on the welcome screen does not match the overall design style. I suggest to just hide it.

## Screenshots
a
b

## Testing strategy
Manually

## Type of change
- ✅ Bug fix (non-breaking change that fixes an issue)

### CODE REVIEW

File: src/gui/MainWindow.cpp [hunk 1]
```diff
@@ -1155,8 +1160,11 @@ void MainWindow::switchToDatabases()
 {
     if (m_ui->tabWidget->currentIndex() == -1) {
         m_ui->stackedWidget->setCurrentIndex(WelcomeScreen);
+        statusBar()->hide();
     } else {
         m_ui->stackedWidget->setCurrentIndex(DatabaseTabScreen);
+        statusBar()->show();
+        statusBar()->setStyleSheet("");
```
Comment (droidmonkey) [inline src/gui/MainWindow.cpp]: We don't set a stylesheet on the statusbar, so this shouldn't be necessary

File: src/gui/MainWindow.cpp [hunk 2]
```diff
@@ -1265,8 +1273,11 @@ void MainWindow::databaseTabChanged(int tabIndex)
 {
     if (tabIndex != -1 && m_ui->stackedWidget->currentIndex() == WelcomeScreen) {
         m_ui->stackedWidget->setCurrentIndex(DatabaseTabScreen);
+        statusBar()->show();
+        statusBar()->setStyleSheet("");
```
Comment (droidmonkey) [inline src/gui/MainWindow.cpp]: same here

File: src/gui/MainWindow.cpp [hunk 3]
```diff
@@ -675,6 +675,11 @@ MainWindow::MainWindow()
 
     restoreConfigState();
     updateMenuActionState();
+
+    // Check the current screen and hide the status bar if it is the WelcomeScreen
+    if (m_ui->stackedWidget->currentIndex() == WelcomeScreen) {
+    statusBar()->hide();
+  }
```
Comment (droidmonkey) [inline src/gui/MainWindow.cpp]: need to fix code format

**Review by droidmonkey — CHANGES_REQUESTED** [review]

**Comment by xboxones1** [comment]
Done

**Comment by droidmonkey** [comment]
Code format is still broken

**Review by droidmonkey — APPROVED** [review]

**Pull request merged**